pkg/controller/configmap/config.go

```diff
@@ -166,6 +156,7 @@ func (adaptor *ConfigMapOc) validateConfigMap(configmap *corev1.ConfigMap) []err
 		errs = append(errs, errors.Wrapf(err, "failed to load ConfigMap"))
 		return errs
 	}
+	appendErrorIfNotNil(&errs, validateConfig(cfg, "coe", "adaptor"))
```

Comment:
Remove this line?

Reply:
I want to add this line to check if coe.adaptor is set or not by FillDefaults function. Jianwei's patch will automatically check env to put adaptaor info in shared status. and then to decide create k8s or ocp4 Adapter instance.

Reply:
I don't understand why we still need to check it as it is set by operator itself?
I thought Jianwei removed checking env

Reply:
I get your point, we indeed don't need validate coe.adatpore anymore, since it's set by operator instead of user.